decidim/decidim
# Add templates support for proposal's answers

#### :tophat: What? Why?
*Please describe your pull request.*
This PR adds the posibility of saving response templates for the proposals.

#### Testing

1. Login as admin, visit admin panel 
2. Click Templates / Proposal Answers / new
3. Fill in the form
4. Visit the chosen proposal component in admin 
5. Click on proposals / answer 
6. Select the template from list
7. Click Save 
8. Visit public site , and see the created answer is displayed on website ( except for not_answered state )

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![image](https://github.com/decidim/decidim/assets/105683/922a369f-fa32-4379-bfa5-238966fac919)
![image](https://github.com/decidim/decidim/assets/105683/1ddacc62-e959-45c9-bc0a-f0163247c98a)
![image](https://github.com/decidim/decidim/assets/105683/666f16a6-4e88-46d8-9c12-571d9c17ba5d)


:hearts: Thank you!
